manifest-latest.json

```diff
@@ -245,9 +245,43 @@
     "dir_name_win": "Bitcoin",
     "repo_url": "https://github.com/bitcoin/bitcoin",
     "versions": [
-      "v0.20.0"
+      "v0.20.0",
+      "v0.20.1"
     ],
-    "xbridge_conf": "bitcoin--v0.20.0.conf",
+    "xbridge_conf": "bitcoin--v0.15.1.conf",
+    "wallet_conf": "bitcoin--v0.16.0.conf"
```

Comment:
According to the README:
```
xbridge_conf | This is the name of the XBridge configuration file within the xbridge-confs folder. The title of this file should be the same as the ver_id in all lowercase.
-- | --
wallet_conf | This is the name of the wallet configuration file within the wallet-confs folder. The title of this file should be the same as the ver_id in all lowercase.
```
But here I see `ver_id` = "bitcoin--v0.20.0" and a non-matching "xbridge_conf": "bitcoin--v0.15.1.conf",
Same w/ wallet_conf.
Renaming wallet_conf & xbridge_conf files in the manifest will, of course, require changes to btc.base.j2 and running `app.py` to create new `{wallet,xbridge}-confs/btc--<ver>.conf` files.

Reply:
Arguably, according to the versioning plan this is correct; versions which don't require config changes use the oldest backwards compatible version. So BTC versions 0.15, 0.16, 0.18, 0.19, 0.20, 0.21, 0.22 can all use the exact same configs. But since we have created separate version groups for each major version it may be less confusing to match the config versions with the coin major version. New commit incoming.